src/anm/animation/element.js

```diff
@@ -709,24 +709,26 @@ Element.prototype.tick = function(dt) {
     }
     var isActive = this.timeline.isActive() && this.modifiers(resultTime, dt);
     if (isActive) {
-        this.each(function(child) {
-            child.tick(dt);
-        });
+        if (this.hasSwitch) {
+            this.each(function(child) {
+                if (child.name == this.switch) {
+                    if (this.justSwitched/* || ((this.getTime() - this.switchBand[0]) > child.timeline.getDuration())*/) child.timeline.reset();
```

Comment:
@skavish It is required to reset child time _once_ every time we overflow it's own lifetime duration. I haven't found nice way to do it for the moment :(